tensorboard/plugins/example/tensorboard_plugin_example/__init__.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A sample plugin to demonstrate dynamic loading."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import textwrap
+
+from tensorboard.plugins import base_plugin
+import werkzeug
+from werkzeug import wrappers
+
+
+class ExamplePlugin(base_plugin.TBPlugin):
+  plugin_name = 'example'
+
+  def __init__(self, context):
```

Comment:
Can't this just be omitted?  Or do you have it there for demonstration purposes?  If the latter, maybe replace `pass` with a comment to that effect.

Reply:
No, it can’t. If you omit this, you get an inscrutable error in the
depths of TensorBoard. I had a patch to make it possible to omit this,
but couldn’t test it due to #2355. Patch:
<https://gist.github.com/wchargin/c6f69767393dc55db653f6daa73b9b91>

Reply:
Let's just add the patch, we can test against an older bazel.

Reply:
Done: #2360 merged.